### DISCUSSION TITLE
uboss (iboss)is not working for me

### DISCUSSION THREAD

**Discussion by Nintendoboi22** [3kh0/ext-remover#622] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
idk if its me but when i open the iboss link and run the code provided nothing happens, when i click the link which is a about:blank link and run the code again it does not work.

**Comment by ghost** [comment]
Currently uboss isn’t working

**Comment by BroJac5246** [comment, marked as answer]
If you have iBoss then I would suggest [Ingot for iBoss](https://github.com/3kh0/ext-remover/discussions/585). It works great.
